### ISSUE TITLE
Tap on error state of chat doesn't try to reload the chat.

### ISSUE THREAD

**Issue by taschik** [GetStream/stream-chat-react-native#396]
**Describe the bug**
If the chat cannot be loaded, for what ever reason, the error state's retry mechanisms doesn't work. A tap on the error message doesn't reload the chat.

**Dev environment info (please complete/provide the following information):**
```
    "stream-chat": "^2.6.0",
    "stream-chat-react-native": "^1.3.4",
```

```
System:
    OS: macOS 10.15.7
    CPU: (12) x64 Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
    Memory: 330.18 MB / 32.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 14.8.0 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.7 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.1, DriverKit 19.0, macOS 10.15, tvOS 14.0, watchOS 7.0
    Android SDK:
      API Levels: 23, 26, 27, 28, 29
      Build Tools: 23.0.1, 25.0.0, 26.0.3, 27.0.3, 28.0.1, 28.0.2, 28.0.3
      System Images: android-24 | Google APIs Intel x86 Atom, android-24 | Google Play Intel x86 Atom, android-29 | Google APIs Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6308749
    Xcode: 12.1/12A7403 - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_221 - /usr/bin/javac
    Python: 3.8.5 - /usr/local/opt/python/libexec/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1
    react-native: 0.63.3 => 0.63.3
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```
Happens on iOS 14.1 Simulator as well as physical device. Tested on iPhone XS.

**To Reproduce**
1. Disconnect from the internet before loading the chat component and you will see the error message.
2. Reconnect to the internet and tab the error message.

**Expected behavior**
The chat would try to load again after tapping the error message.

**Screenshots**

![96921895-fe8e7980-1463-11eb-9aa4-a266ef29296c](https://user-images.githubusercontent.com/351623/97054390-8eeabe00-1539-11eb-9276-8b61e7ad8715.png)

gz#6842

**Comment by vishalnarkhede** [comment]
Thanks for bringing this up @taschik. We will fix this soon on 2.x.x :)

**Comment by vishalnarkhede** [comment]
Hey @taschik , this has been fixed in `2.0.2`. Please reopen the issue if it doesn't work for you :)
